.github/workflows/container-image.yaml

```diff
@@ -60,12 +56,56 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Build and push
+      - name: Docker metadata - Main
+        id: meta1
```

Comment:
Can we use more descriptive names for `metaX`?

Reply:
Changed in latest commit